docs/network protocol.md

```diff
@@ -702,14 +702,17 @@ GameData is a **dict** but contains these keys and values. It's broken out into
 | checksum            | str            | A checksum hash of this game's data.                                                                                          |
 
 ### Tags
-Tags are represented as a list of strings, the common Client tags follow:
-
-| Name       | Notes                                                                                                                                                                                                              |
-|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| AP         | Signifies that this client is a reference client, its usefulness is mostly in debugging to compare client behaviours more easily.                                                                                  |
-| DeathLink  | Client participates in the DeathLink mechanic, therefore will send and receive DeathLink bounce packets                                                                                                            |
-| Tracker    | Tells the server that this client will not send locations and is actually a Tracker. When specified and used with empty or null `game` in [Connect](#connect), game and game's version validation will be skipped. |
-| TextOnly   | Tells the server that this client will not send locations and is intended for chat. When specified and used with empty or null `game` in [Connect](#connect), game and game's version validation will be skipped.  |
+Tags are represented as a list of strings, the common client tags follow:
+
+| Name       | Notes                                                                                                                             |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| AP         | Signifies that this client is a reference client, its usefulness is mostly in debugging to compare client behaviours more easily. |
+| DeathLink  | Client participates in the DeathLink mechanic, therefore will send and receive DeathLink bounce packets                           |
+| HintGame\* | Special tag indicating the client is a hint game, made to send hints instead of locations. Special join/leave message.            |
```

Comment:
```suggestion
| HintGame\* | Indicates the client is a hint game, made to send hints instead of locations. Special join/leave message.                         |
```
Maybe?

Reply:
The edit should be made for the other tags as well, but I do agree with the edit

Reply:
Should be made for the other tags as well. I agree with the edit/change